pull_request_test.rb

```diff
@@ -2,9 +2,30 @@
 #
 # Ad-hoc script for updating a pull request using our service.
 #
+###
 # Usage:
 #
-#   $ OAUTH_TOKEN="..." bundle exec ruby pull_request_test.rb
+#   $ script SERVER REPO PR OAUTH_TOKEN SERVICE_TOKEN
```

Comment:
To unpack this a little bit:

The script should work as-is with either version of the services gem, so no change should be needed. The idea is it's here with example values that you modify locally to test whatever it is you need to test. If we checked in those modifications every time, the script would see unnecessary churn.

We did identify that the `OAUTH_TOKEN` variable the script comment shows is a little mysterious, so I could see a change to further describe how to find a suitable value for that variable being valuable here.